packages/neuron-ui/src/components/PricePanel/index.tsx

```diff
@@ -107,11 +115,35 @@ const PricePanel: React.FunctionComponent<PricePanelProps> = ({ price, field, on
     [priceArray, feeValuesArray]
   )
 
+  const batchGenerateExperimental = useCallback(async () => {
+    if (experimental && experimental?.params) {
+      const generator = getGenerator(experimental.params?.sendType)
+      try {
+        const requestArray = priceArray.map(itemPrice => generator({ ...experimental.params, feeRate: itemPrice }))
+        const allPromiseResult = await Promise.allSettled(requestArray)
+
+        const resList = allPromiseResult.map((batchItem, index: number) => ({
+          feeRateValue: priceArray[index],
+          // @ts-ignore
+          feeValue: batchItem.value.result?.fee,
+        }))
+
+        setFeeRateValueArray(resList)
+      } catch (error) {
+        // @ts-ignore
```

Comment:
I think there should be an explanation for any place that swallows errors, even if it's just saying "this error is not important".